src/Attributes/src/Internal/Instantiator/DoctrineInstantiator.php

```diff
@@ -136,11 +135,6 @@ private function propertyNotFound(\ReflectionClass $attr, string $name, ?\Reflec
         return new SemanticAttributeException($message);
     }
 
-    /**
-     * @param \ReflectionClass $class
-     * @return string
-     */
-    #[Pure]
     private function getAvailablePropertiesString(\ReflectionClass $class): string
```

Comment:
this function is not pure.

Reply:
( note: don't trust phpstorm when it says that a function is pure, phpstorm purity checker is completely broken )

Reply:
yea, thx!